### ISSUE TITLE
loss is always nan 

### ISSUE THREAD

**Issue by mwindowshz** [pierluigiferrari/ssd_keras#74]
Hi
Sorry I am an expert.
Using latest version from github.

I am trying to run ssd 300 training. using PASCAL VOC files
There is a problem 
loss is nan. all the time!

what Is wrong?

> 
>  39/518 [=>............................] - ETA: 8:37 - loss: nan
>  40/518 [=>............................] - ETA: 8:33 - loss: nan
>  41/518 [=>............................] - ETA: 8:28 - loss: nan
>  42/518 [=>............................] - ETA: 8:25 - loss: nan
>  43/518 [=>............................] - ETA: 8:21 - loss: nan

Keras                     2.1.4

tensorflow-gpu            1.5.0                   
tensorflow-tensorboard    1.5.1              

Is this a problem in my setup? or the code?
I use Keras on other project , and training works fine.

Thanks
m.

**Comment by pierluigiferrari** [comment]
You already posted in #55 (15 days ago) that you trained SSD300 on Pascal VOC and got a decreasing floating point loss. The code hasn't changed since (last commit 17 days ago), so you already know that the code is working.

**Comment by mwindowshz** [comment]
Yes, I am working on a different computer, and now there is this problem.
Since older attempts to run the training did not work,  I tried to work with a different setup. and start again.

I guess the problem of nan is related to some library that is installed.

sorry to bother you.

**Comment by mwindowshz** [comment]
**Fixed the prblem**
I created a new conda enviroment
installed older tensorflow, keras , gpu verison using cuda 8  (from anconda cloude - maybe this https://anaconda.org/anaconda/keras-gpu
added system path to the conda environment, so Microsoft visual 2015 and cuda 8 dlls were in path.

Before I had cuda 9.1 installed, with tensorflow using a version compatible with cuda 9.1  
My guess some files were incompatible.
now it started working.

Hope this helps any other user that has this problem.

Now I would wait, to check if learning would succeed.

**Comment by pierluigiferrari** [comment]
Thanks for sharing this. I'm planning to update to CUDA 9 and TensorFlow 1.5 myself soon, so let's see if I'll run into similar problems.
